pkg/dies/cartographer/v1alpha1/zz_generated.die.go

```diff
@@ -759,6 +889,14 @@ func (d *RealizedResourceDie) DieFeedPtr(r *cartographerv1alpha1.RealizedResourc
 	return d.DieFeed(*r)
 }
 
+// DieFeedRawExtension returns the resource managed by the die as an raw extension.
+func (d *RealizedResourceDie) DieFeedRawExtension(raw runtime.RawExtension) *RealizedResourceDie {
```

Comment:
why is dies updated in this PR

Reply:
Not sure, I just ran 
`make install`

And that happened. Same in the other PR. Will revert the changes.